jira/client.py

```diff
@@ -3984,7 +4043,18 @@ def permissionschemes(self):
         return data["permissionSchemes"]
 
     @lru_cache(maxsize=None)
-    def issuesecurityschemes(self):
```

Comment:
Unfortunately, as this is an API that is widely used, we cannot simply change these public methods.
If you are keen to make this change, my suggestion would be to: make a separate pull request where you deprecate this existing function name, and make it call the new function name